src/Utils/View.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Masterclass\Utils;
+
+use Masterclass\Exceptions\BadViewDataException;
+
+class View
+{
+    /**
+     * @param string $__view__ The name of the view
+     * @param array $__data__ an array of data
+     * @return mixed
+     * @throws BadViewDataException
+     */
+    public static function make($__view__, $__data__ = [])
```

Comment:
Statics are generally considered harmful in this case. I'll share why this weekend or during office hours.